client/reader/stream/customer-council-banner.jsx

```diff
@@ -0,0 +1,56 @@
+import { Reader, SubscriptionManager } from '@automattic/data-stores';
+import Banner from 'calypso/components/banner';
+import { navigate } from 'calypso/lib/navigate';
+
+export const CustomerCouncilBanner = ( { translate } ) => {
+	const CUSTOMER_COUNCIL_P2_URL = 'https://readercouncilgeneral.wordpress.com/';
+	const CUSTOMER_COUNCIL_P2_ID = '237686330';
+
+	const { mutate: subscribe, isIdle: notActivelySubscribing } =
+		SubscriptionManager.useSiteSubscribeMutation();
+
+	const {
+		data: p2,
+		isFetched: checkedAlreadySubscribed,
+		isFetching: checkingAlreadySubscribed,
+	} = Reader.useReadFeedSiteQuery( Number( CUSTOMER_COUNCIL_P2_ID ) );
+	const alreadySubscribed = p2?.is_following;
+
+	const hideBanner = ( alreadySubscribed && checkedAlreadySubscribed ) || checkingAlreadySubscribed;
```

Comment:
Not a critique on your PR. But I am very interested to see how much response we get by showing this to all users at once! 

If we did have a desire to limit how many users see this at first, we could check some variation of `userId % 100 < X` where X is the percent we do or don't want to show it to, and ramp up the number as needed.  I think we had that idea initially in January before we got pulled away from thins but have decided to just show this to everyone for a shorter amount of time instead?

Long story short, im just interested to see the response from this! 😅

Reply:
Do we have a mechanism to do slow rollouts like you're describing?

Reply:
Not off the top of my head. We would probably do a check using something like `userId % 100 < X`, and just incrementally push PRs to update the value of X to include more users. If we really wanted we could set that number to be based on date, fetched from an endpoint, or there are probably some other options as well in that kind of case.

I think I was only involved in 1 or 2 "slow rollouts" like this and we just updated the value in later PRs when we wanted to increase the number.